packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -2761,7 +2764,7 @@ export class IntervalCollection<TInterval extends ISerializableInterval>
 				oldSeg?.localRefs?.addLocalRef(oldInterval.end, oldInterval.end.getOffset());
 			}
 			this.localCollection.add(interval);
-			this.emitChange(interval, oldInterval as TInterval, true, op);
+			this.emitChange(interval, oldInterval as TInterval, true, false, op);
```

Comment:
general comment on the PR description:

I would expect that undo-redo does legitimately *sometimes* want to pay attention to slides and put them on an undo/redo stack. Specifically, the client that performed the text removal should also probably put a "change the interval back to where it was" operation on its undo-redo stack. With the way things are ordered I can see how that would be difficult (slide happens on ack of removal, at which point the removal may no longer be at the top of the stack, but we can only slide back after the removal gets undone). but figured it was worth calling out.

Reply:
The segment remove revertible already handles the interval endpoints within it that will slide. So it will go to its original position. Similarly for add and change intervals where the containing segment becomes removed, although I think we may have a bug there that @jzaffiro is looking at where you can't move an interval endpoint into a removed segment (probably just need to move it to the place where it will slide).